plugins/woocommerce/includes/class-wc-install.php

```diff
@@ -169,6 +169,10 @@ class WC_Install {
 			'wc_update_600_migrate_rate_limit_options',
 			'wc_update_600_db_version',
 		),
+		'6.1.0' => array(
```

Comment:
> Scheduled for WooCommerce 6.2

Given this note, should we adjust to `'6.2.0'` (and if so also adjust `wc_update_610_db_version()`)?

Reply:
Yep, makes sense. Fixed.

Reply:
Yep, makes sense. Fixed.